tests/framework_django/test_wildcard_filters.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import django
+import pytest
+from testing_support.fixtures import collector_available_fixture, django_collector_agent_registration_fixture
+from testing_support.validators.validate_code_level_metrics import validate_code_level_metrics
+from testing_support.validators.validate_transaction_count import validate_transaction_count
+from testing_support.validators.validate_transaction_metrics import validate_transaction_metrics
+
+DJANGO_VERSION = tuple(map(int, django.get_version().split(".")[:2]))
+
+if DJANGO_VERSION[0] < 3:
+    pytest.skip("support for asgi added in django 3", allow_module_level=True)
+
+# Import this here so it is not run if Django is less than 3.0.
+from testing_support.asgi_testing import AsgiTest  # noqa: E402
+
+_default_settings = {
+    "package_reporting.enabled": False,  # Turn off package reporting for testing as it causes slow downs.
+    "transaction_tracer.explain_threshold": 0.0,
+    "transaction_tracer.transaction_threshold": 0.0,
+    "transaction_tracer.stack_trace_threshold": 0.0,
+    "debug.log_data_collector_payloads": True,
+    "debug.record_transaction_failure": True,
+    "debug.log_autorum_middleware": True,
+    "instrumentation.django_middleware.enabled": True,
+}
+
+wildcard_exclude_specific_include_settings = (
```

Comment:
Can we get some more tests here:
* there aren't *'d paths.
* there is only an exclude rule
* there is only an include rule
* there are no dots in the paths
* there are only dots in the include path
* there are only dots in the exclude path
* there is a path with a dot (no *) and a path with no dots and only a *
* case sensitive test
There's probably some additional cases I'm not thinking of?

I want to be really thorough when we test these cause there's a lot of complex logic there and I don't want to miss some corner case bug in the implementation. (I don't see any but it might be hard to catch without thorough testing.)